### ISSUE TITLE
Sending a large detached list of simple objects breaks down due to server timeouts or size limits

### ISSUE THREAD

**Issue by didimitrie** [specklesystems/speckle-sharp#2748]
TL;DR: we need a better way to handle large lists of detachable objects in side core and py. 

https://speckle.community/t/failing-to-send-layer-from-qgis-large-geojson-with-120k-entities/6301/4

When investigating the issue above with @KatKatKateryna: it's a legitimate connector/object model need. 

TODO: 
Create a simple test case experiment with a list of 120k+ points and send it to the server. 

Possible future "fast" resolutions: 
- increase timeout limits on server
- increase server max object size

Longer term, we need to think of handling this better architecturally around connector <> core <> transport <> etc.

**Comment by KatKatKateryna** [comment]
The example commit with 1 point element instead of 124 000 : https://speckle.xyz/streams/17b0b76d13/commits/06b3b62907

**Comment by didimitrie** [comment]
Note from @jsdbroughton and @teocomi: possibly related to this: https://github.com/specklesystems/speckle-sharp/issues/2360#issuecomment-1550016789

**Comment by bimgeek** [comment]
Closing this ticket as we are moving to Jira to reduce noise. You can access it there.
